src/python_testing/TC_EVSE_2_1.py

```diff
@@ -0,0 +1,90 @@
+
+import logging
+
+import chip.clusters as Clusters
+from chip.interaction_model import Status
+from matter_testing_support import MatterBaseTest, async_test_body
+from mobly import asserts
+
+# List of attributes tests in the following format: ['Attribute name',read/write, Value to Write, ExpectedValue, Test step]
+Attribute_names = [
+    ['State', 'r', None, None],
+    ['SupplyState', 'r', None, None],
+    ['FaultState', 'r', None, None],
+    ['ChargingEnabledUntil', 'r', None, None],
+    ['DischargingEnabledUntil', 'r', None, None],
+    ['CircuitCapacity', 'r', None, None],
+    ['MinimumChargeCurrent', 'r', None, None],
+    ['MaximumChargeCurrent', 'r', None, None],
+    ['MaximumDischargeCurrent', 'r', None, None],
+    ['UserMaximumChargeCurrent', 'rw', 20000, 20000],
+    ['UserMaximumChargeCurrent', 'rw', 90000, 20000],    # Constraints test
+    ['UserMaximumChargeCurrent', 'rw', -90000, 20000],    # Constraints test
+    ['RandomizationDelayWindow', 'rw', 699, 699],
+    ['RandomizationDelayWindow', 'rw', 90000, 699],      # Constraints test
```

Comment:
Does the order of this table matter then? ie, you write the out of range value, but expect the previous value. Is that because you're truncated to the range? Or because you're keeping the value from the previous write?